config/initializers/form_error.rb

```diff
@@ -0,0 +1,49 @@
+# Insert model validation error messages after the input elements.
+#
+# Add this file to config/initializers/form_errors.rb
+#
+# You can configure these options here.
+
+config = {
+  error: 'error',
+  invalid: 'invalid',
+  template: '<span class="{error}"><br>{message}</span>'
+}
+
+ActionView::Base.field_error_proc =
```

Comment:
I'm wondering why this was necessary. Don't we already have any error for form inputs?

Reply:
Yes it's just a proposal to have error messages under the inputs but maybe too much in one branch. I rollback.

Reply:
Yeap I agree: should be another PR then. Thanks!